Fix:
error: implicitly declaring library function 'strerror' with type 'char *(int)' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'usleep' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- die.c.orig	2005-03-18 06:23:50.000000000 -0600
+++ die.c	2022-05-14 04:57:17.000000000 -0500
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <errno.h>
 #include <stdarg.h>
 #include "config.h"
--- queue.c.orig	2000-08-14 12:24:54.000000000 -0500
+++ queue.c	2022-05-14 04:58:38.000000000 -0500
@@ -12,6 +12,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <pthread.h>
+#include <unistd.h>
 #include "config.h"
 #include "queue.h"
 #include "mgzip.h" /* for the GIVE_IT_UP macro */
